cloud/page-index/page-index.js

```diff
@@ -53,6 +53,10 @@ module.exports = [
       {
         title: "Account management",
```

Comment:
capitalize M so Account Management?

Reply:
We use sentence case, so it's correct with a lowercase M.